test/spec/transformDimension.spec.ts

```diff
@@ -29,47 +33,61 @@ describe('transform dimension', () => {
         transformDimension({
           type: 'typography',
           value: {
-            fontSize: '4',
+            fontSize: '4 8',
```

Comment:
Can we create 2 separate expects in this test or a separate test, one that checks for single value and one that checks for multi value? I wanna make sure we keep also covering the single case. Same thing for the shadow test, keep a test that covers single value offsetX/offsetY props etc.